aes256.rs

```diff
@@ -0,0 +1,338 @@
+// Import hacspec and all needed definitions.
+use hacspec::prelude::*;
+// TODO: move to hacspec_imports if we want to use it!
+use contracts::*;
+// I need 32 bytes for a AES256 key CHANGED FROM 16 to 32
+const BLOCKSIZE: usize = 32;
+// CHANGED from 12 to 28, not sure if it is right
+const IVSIZE: usize = 12;
```

Comment:
You can generalise to other IV sizes. But it's easier to stay with 12 for now and do the IV change in a separate PR.